e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -1431,6 +1460,52 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       clickLineChartPoint();
     });
 
+    it("allows opening custom URL destination that is not a Metabase instance URL using link (metabase#33379)", () => {
```

Comment:
It's virtually impossible to test metabase running in subpath, I've tried a few approaches, but didn't work out.

This test however tests that our new code that matches subpaths works. It tests that even if the origin is the same but the pathnames are different, Metabase should navigate using a link (refresh page).